scripts/actions/generate_new_version_code.sh

```diff
@@ -25,6 +25,6 @@ echo "NEW_TAG=$new_tag" >> $GITHUB_ENV # Setting this for use later
 echo "Updating CurrentVersionCode by 1 and changing file versionCode"
 
 let "currentVersionCode=currentVersionCode+1" # Bumping versionCode By one
-sed -i '' "s/\(versionCode[[:space:]]*=[[:space:]]*\)[0-9]*/\\1${currentVersionCode}/" $GITHUB_WORKSPACE/app/app.gradle.kts
+sed -i "s/\(versionCode[[:space:]]*=[[:space:]]*\)[0-9]*/\\1${currentVersionCode}/" $GITHUB_WORKSPACE/app/app.gradle.kts
```

Comment:
@ManavWednesday Can we add a comment on top of this line state that the `''` are required on macOS and not on linux. That way anyone looking at this will know what to do in case of errors.

Reply:
Cool